src/api/app/controllers/source_project_controller.rb

```diff
@@ -0,0 +1,209 @@
+class SourceProjectController < SourceController
+  validate_action update_project_meta: { request: :project, response: :status }
+  validate_action show_project_meta: { response: :project }
```

Comment:
There is one [before action](https://github.com/openSUSE/open-build-service/pull/5085/files#diff-ef87c76ec3b8d2870b8ddcf9fb2389c1R21) that you didn't move over.
Was that intentionally or did you overlook it? I am asking because with the :except / :only keys things can easily get mixed up

Reply:
Becase SourceProjectController is a subclass from SourceController, I didn't see a reason to touch it now.